optax/_src/loss.py

```diff
@@ -358,3 +358,33 @@ def loop_body(prev, x):
   per_seq_loss = -jnp.einsum('bn,bn->b', logalpha_phi_last, one_hot)
 
   return per_seq_loss
+
+
+def kl_div_loss(predictions: chex.Array,
+                targets: chex.Array,
+                log_targets: bool = False) -> chex.Array:
+  """Computes the Kullback-Leibler divergence (relative entropy) loss.
+
+  Measures the information gain achieved if target probability distribution
+  would be used instead of predicted probability distribution.
+
+  References:
+    [Kullback, Leibler, 1951](https://www.jstor.org/stable/2236703)
+
+  Args:
+    predictions: Probabilities of predicted distribution with shape [..., dim].
+    Expected to be in the log-space to avoid underflow.
```

Comment:
In theory pylint should complain here ... there should be 2 white spaces before "Expected" so that it's indented in the section of "predictions". Once you have synced we can run the testing workflow again which should have flagged this (the workflow broke shortly before you committed but we have since fixed it).